startvscode.cmd

```diff
@@ -0,0 +1,32 @@
+@ECHO OFF
+SETLOCAL
+
+:: This command launches a Visual Studio code with environment variables required to use a local version of the .NET Core SDK.
+
+:: This tells .NET Core to use the same dotnet.exe that build scripts use
+SET DOTNET_ROOT=%~dp0.dotnet
+SET DOTNET_ROOT(x86)=%~dp0.dotnet\x86
+
+:: This tells .NET Core not to go looking for .NET Core in other places
+SET DOTNET_MULTILEVEL_LOOKUP=0
+
+:: Put our local dotnet.exe on PATH first so Visual Studio knows which one to use
+SET PATH=%DOTNET_ROOT%;%PATH%
+
+:: Sets TFW for Visual Studio Code usage
+SET TARGET=net6.0
```

Comment:
This is the only place where we need to change `TargetFramework` when it's gets changed next time. All child `startvscode.cmd` file will read this value!

Reply:
Great! Were you able to figure out a way to incorporate this into the target framework referenced in the `launch.json` files?

Reply:
Yes Safia, `launch.json` will use this `env` value. As you can see below. 
https://github.com/dotnet/aspnetcore/blob/c7a5c9e68f2a43b669242aad4028729cdce1ab98/src/Mvc/.vscode/launch.json#L9

So, at the time of the next release, we just need to change the `TFW` in `startvscode.cmd` file.